### PR TITLE
Added ability to open checking account

### PR DESCRIPTION
**Pull request by abenzzine10**
- Added a unique 6 digits account number field to accounts to use later when transferring money
- Added ability to open a checking account
- Added a class to display the account menu
- Added the ability to display accounts opened by the current user

### CODE REVIEW

File: src/UserMenu.java [hunk 1]
```diff
@@ -70,18 +68,36 @@ public boolean createSavingsAccount() {
...
 	
-	/* TODO: Display list of accounts for user to select from and 
-	 * allow user to pick one to display the account menu
+	/**
+	 * Displays the accounts of the current user and asks them which account they want to select
 	 */
-	public boolean selectAccount() {
-		System.out.println("TO BE IMPLEMENTED: Select Account");
-		return true;
+	public void selectAccount() {
```
Comment (andrewmitra) [inline src/UserMenu.java:93]: Can you separate this into smaller functions?

Comment (abenzzine10) [reply src/UserMenu.java:93]: Sure

**Review by andrewmitra — CHANGES_REQUESTED** [review]
Can you refactor the one method and add unit test for adding account for a user

File: src/UserMenu.java (hunk 1 above)
Comment (abenzzine10) [reply src/UserMenu.java:93]: Done

**Review by andrewmitra — APPROVED** [review]

**Pull request merged**